lib/commonBindings.js

```diff
@@ -92,10 +92,7 @@ function guessType(attribute, device) {
     }
```

Comment:
CHANGES_NEXT_RELEASE entry is needed. Something like this:

```
- Remove: NGSI-v1 specific behaviours
```

Reply:
Fixed in https://github.com/telefonicaid/iotagent-json/pull/533/commits/fd29166d7c8e9df6b9a8ffad02b335ef63d1246f